src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -23,7 +23,16 @@ public abstract class RawImageUwbClientInputHandler : RawImageUwbClientManager,
         /// </summary>
         [Tooltip("The input handler to use")]
         public WebBrowserInputHandler inputHandler;
-        
+
+        [Tooltip("Disable the usage of mouse and keyboard")]
+        public bool disableAllControls;
+
+        [Tooltip("Disable the usage of mouse")]
+        public bool disableMouse;
```

Comment:
Add the tooltip's text to a XML doc comment as well (See the `inputHandler` above as an example)